### ISSUE TITLE
verticaltab: collapsible aria-labelling

### ISSUE THREAD

**Issue by justinclary** [pluralsight/design-system#1747]
### Expected behavior

* When the vertical tab is collapsible, the aria-label should announce whether or not it is collapsed, and shouldn't read the up/down caret

### Actual behavior

* The screen reader reads the whole content of the vertical tab including the caret, and doesn't provide the collapsible state of the vertical tab.

### Steps to repro

1. Use a screen reader
2. navigate to the collapsible vertical tab
3. screen reader announces content of button and caret icon

### Related packages

- verticaltabs@7.1.4

### Environment

- OS - MacOS 11.3 (Big Sur)
- Browser version - Chrome 90.0.4430.93
